Add UKLWC

beckyjackson opened the pull request:
Mappings approved via email

jamesaoverton reviewed (changes requested):
The data dictionary content looks good.

What happened to `data/ihcc-mapping-suggestions-zooma.tsv`?

The `data/ols-config.yaml` was listing the dictionaries alphabetically, but now it's scrambled. Can you make it consistent?

beckyjackson commented:
> What happened to data/ihcc-mapping-suggestions-zooma.tsv?

It gets rebuilt randomly every time. I made a fix to the python script to sort this by IRI, but I also messaged Nico to make sure this is OK.

> The data/ols-config.yaml was listing the dictionaries alphabetically, but now it's scrambled. Can you make it consistent?

Fixed. I added a sort step in the script.
